src/control/settings/PageTemplateSettings.cpp

```diff
@@ -3,6 +3,7 @@
 #include "control/xojfile/SaveHandler.h"
 
 #include <sstream>
+#include <control/pagetype/PageTypeHandler.h>
```

Comment:
use dqoutes, group it to the others alphabetically and move the STL import to the top